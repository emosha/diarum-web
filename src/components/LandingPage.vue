<template>
  <div class="landing-page">
    <Hero />
    <Features />
    <Footer />
  </div>
</template>

<script lang="ts">
import { Component, Prop, Vue } from 'vue-property-decorator';
import Hero from '@/components/Hero.vue';
import Navbar from '@/components/Navbar.vue';
import Features from '@/components/Features.vue';
import Footer from '@/components/Footer.vue';
import Wave from '@/components/shared/Wave.vue';

@Component({
  components: {
    Hero,
    Navbar,
    Wave,
    Features,
    Footer,
  },
})
export default class LandingPage extends Vue {}
</script>

<style lang="scss">
  // @media (max-width: 1199.98px) {
  //   .arrow {
  //     display: none;
  //   }
  // }
</style>
