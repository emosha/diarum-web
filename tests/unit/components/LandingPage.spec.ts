import { shallowMount } from '@vue/test-utils';
import LandingPage from '@/components/LandingPage.vue';

describe('LandingPage.vue', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(LandingPage);
    expect(wrapper).toMatchSnapshot();
  });
});
