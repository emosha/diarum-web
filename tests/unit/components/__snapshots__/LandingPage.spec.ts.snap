// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage.vue renders correctly 1`] = `
<div class="landing-page">
  <hero-stub></hero-stub>
  <features-stub></features-stub>
  <footer-stub></footer-stub>
</div>
`;
